### PR TITLE
Webstorm now supports Prettier

### PR DESCRIPTION
**Pull request by tusharmath**
Webstorm now supports prettier!

### CODE REVIEW

File: docs/webstorm.md [hunk 1]
```diff
@@ -3,6 +3,9 @@ id: webstorm
 title: Webstorm Setup
 ---
 
+## Version 2018.1 and above
+Webstorm now comes with prettier integration by default. For older versions checkout the details below.
```
Comment (j-f1) [inline docs/webstorm.md]: > For older versions, please follow the instructions below.

?

**Comment by azz** [comment]
We should mention it in the landing page too.

**Comment by j-f1** [comment]
@azz Can you open an issue so we can track this separately?

**Comment by azz** [comment]
#4037

**Pull request merged**